newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js

```diff
@@ -6,11 +6,14 @@ import ObjectSelector from '../../ObjectsList/ObjectSelector';
 import { type ParameterFieldProps } from './ParameterFieldCommons';
 import { Trans } from '@lingui/macro';
 import { nameAndIconContainer } from '../EventsTree/ClassNames';
+import InAppTutorialContext from '../../InAppTutorial/InAppTutorialContext';
 
 export default class ObjectField extends React.Component<
   ParameterFieldProps,
   {||}
 > {
+  static contextType = InAppTutorialContext;
```

Comment:
Not pretty but because it's faster it's fine for now!

Reply:
Yes but this `super(props)` afraid me a little